tests/letstest/multitester.py

```diff
@@ -99,11 +99,9 @@
 # Globals
 #-------------------------------------------------------------------------------
 BOULDER_AMI = 'ami-072a9534772bec854' # premade shared boulder AMI 18.04LTS us-east-1
-LOGDIR = "" #points to logging / working directory
-# boto3/AWS api globals
-AWS_SESSION = None
-EC2 = None
+LOGDIR = "letest-%d"%int(time.time()) #points to logging / working directory
 SECURITY_GROUP_NAME = 'certbot-security-group'
+SENTINEL = None #queue kill signal
```

Comment:
I see no code here that change `SENTINEL` value to something else than `None`. Is it really used ?

In parallel, setting a global variable to `None` here indicate a mutable global state. I think it can be improved, by creating it lazily during the multitester execution.

Reply:
You're right the value of `SENTINEL` is never changed.

`SENTINEL` is used to tell client processes that their work is done. See https://github.com/certbot/certbot/pull/7057/files#diff-821c98fdcbcf065345fc89980bc6609eR343.

The way this works is the parent process loads the queue with a bunch of client instances that need tests run on them at https://github.com/certbot/certbot/pull/7057/files#diff-821c98fdcbcf065345fc89980bc6609eR542. The child processes work through this list until it reaches the `SENTINEL` value at which point they exit.

`SENTINEL` is a global constant for this purpose. For what it's worth, this aspect never changed. `SENTINEL` is currently a global constant that works that way in `master`, I just moved setting it's value to the top of the script so it is outside of the `main()` function.

Reply:
OK I see! So it is irrelevant for this PR, as it does not change a logic that was working the similarly before.